src/Stryker.Core/Stryker.Core/Mutants/MutantPlacer.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using System.Collections.Generic;
+
+namespace Stryker.Core.Mutants
+{
+    public static class MutantPlacer
+    {
+        public static IfStatementSyntax PlaceWithIfStatement(StatementSyntax original, StatementSyntax mutated, int mutantId)
+        {
+            // place the mutated statement inside the if statement
+            return SyntaxFactory.IfStatement(
+                condition: GetBinaryExpression(mutantId),
+                statement: SyntaxFactory.Block(mutated),
+                @else: SyntaxFactory.ElseClause(SyntaxFactory.Block(original)))
+                // Mark this node as a MutationIf node. Store the MutantId in the annotation to retrace the mutant later
+                .WithAdditionalAnnotations(new SyntaxAnnotation("MutationIf", mutantId.ToString()));
+        }
+
+        public static SyntaxNode RemoveByIfStatement(SyntaxNode node)
+        {
+            if (node is IfStatementSyntax ifStatement)
+                return ifStatement.Else.Statement;
+            else return null;
+        }
+
+        public static ConditionalExpressionSyntax PlaceWithConditionalExpression(ExpressionSyntax original, ExpressionSyntax mutated, int mutantId)
+        {
+            // place the mutated statement inside the if statement
+            return SyntaxFactory.ConditionalExpression(
+                condition: GetBinaryExpression(mutantId),
+                whenTrue: mutated,
+                whenFalse: original)
+                // Mark this node as a MutationConditional node. Store the MutantId in the annotation to retrace the mutant later
+                .WithAdditionalAnnotations(new SyntaxAnnotation("MutationConditional", mutantId.ToString()));
+        }
+
+        public static SyntaxNode RemoveByConditionalExpression(SyntaxNode node)
+        {
+            if (node is ConditionalExpressionSyntax conditional)
+                return conditional.WhenFalse;
+            else return null;
+        }
+
+        private static ExpressionSyntax GetBinaryExpression(int mutantId)
```

Comment:
This function could do with some documentation. Maybe use named parameters at the very least. It would also help if the objects were constructed separately before combining and returning. Right now this is 20 lines of inline object construction.